packages/hooks/src/hooks.spec.js

```diff
@@ -45,35 +45,32 @@ describe('packages/hooks', () => {
   describe.skip('useMutationObserver', () => {}); //eslint-disable-line jest/no-disabled-tests
 
   describe('useElementNode', () => {
-    let count = 1;
+    let count = 0;
 
     function TestUseElementNode() {
       const [refEl, setRefEl] = useElementNode();
       useEffect(() => {
-        count += 1;
+        ++count;
```

Comment:
I'd prefer we keep it as `count += 1` for readability. Prefix or postfix increments always confuse me.

Reply:
+1

Reply:
👍